docs/policies/policies.md

```diff
@@ -0,0 +1,515 @@
+
+### Policy Template Example with @context: `"odrl": "http://www.w3.org/ns/odrl/2/"`
+
+There is another possibility of representation: in the example above, `"@context": "http://www.w3.org/ns/odrl.jsonld"` was defined, so it is not necessary to always write `odlr:xy`. If this is not defined, the policy must look like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "@type": "PolicyDefinitionRequest",
+  "@id": "{{POLICY_ID}}",
+  "policy": {
+    "@type": "odrl:Set",
+    "odrl:permission": [
+      {
+        "odrl:action": {
+          "@id": "odrl:use"
+        },
+        "odrl:constraint": [
+          {
+            "@type": "odrl:LogicalConstraint",
+            "odrl:and": [
+              {
+                "odrl:leftOperand": {
+                  "@value": "<field>"
+                },
+                "odrl:operator": {
+                  "@id": "odrl:eq"
+                },
+                "odrl:rightOperand": "<value>"
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+### Policies & Verifiable Credenitals (VC)
+
+#### General Information
+
+Catena-X uses policies to determine access to and use of data. The policies refer to verifiable credentials (VC) that are stored in the Managed Identity Wallet (MIW). Catena-X uses the principle of self-sovereign identity (SSI) here.
+
+ℹ️ Detailed Information about SSI, VC and MIW can be found here:
+
+- [https://github.com/eclipse-tractusx/ssi-docu](https://github.com/eclipse-tractusx/ssi-docu)
+- [https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-2](https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-2) (for release 23.09)
+- [https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-3](https://github.com/eclipse-tractusx/ssi-docu/tree/main/docs/architecture/cx-3-3) (for release 23.12)
+
+The key architectural principle underlying this specification is that policy definitions must be decoupled from their corresponding VC schema. Namely, the specific **constraints** and shape of the VC schema must not be reflected in the policy definition. This allows VC schemas to be altered without impacting policy definitions.
+
+#### Membership Constraint Example
+
+The main difference between the various Catena-X policies is their `constraints`. For this reason, only the constraint examples are shown below instead of always showing the entire policy.
+
+To understand what the connection between policy and VC looks like, let's take a look at the following example ([Source](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/edc/policy.definitions.md#1-membership-constraint)):
+
+```json
+{
+    "constraint": {
+    "leftOperand": "Membership",

```

Comment:
From my perspective Membership BPN should NOT be usage, but access policy. Not sure if this is worth a note here because it seems it is not clear from the context.